xbmc/TextureCacheJob.cpp

```diff
@@ -32,7 +32,7 @@
 #include "FileItem.h"
 #include "music/MusicThumbLoader.h"
 #include "music/tags/MusicInfoTag.h"
-#if defined(HAS_OMXPLAYER)
+#if defined(TARGET_RASPBERRY_PI)
```

Comment:
kodi won't build with a mixture of HAS_OMXPLAYER, HAVE_OMXLIB, HAS_MMAL defined/undefined so I have no objections to replacing them with a single define. I don't believe we are losing any functionality.
Ideally all the combinations of these options would be tested and fixed up, but that isn't a trivial job.